python/py-pkgutil_resolve_name/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-pkgutil_resolve_name
+version             1.3.10
+revision            0
+categories-append   devel
+license             MIT
+supported_archs     noarch
+
+python.versions     37 38
+
+maintainers         nomaintainer
+
+description         A backport of Python 3.9’s pkgutil.resolve_name.
+long_description    {*}${description}
+
+homepage            https://github.com/graingert/pkgutil-resolve-name
+
+checksums           rmd160  d63e02f9beb87d70e1f9c95076176233eac639cd \
+                    sha256  357d6c9e6a755653cfd78893817c0853af365dd51ec97f3d358a819373bbd174 \
+                    size    5054
+
```

Comment:
this should build using PEP517 (untested, but this should work I think):
```
python.pep517 yes
python.pep517_backend flit
```

Reply:
After that addition to the Portfile, `port install py38-pgkutil_resolve_name` triggers the automatic installation of `py38-flit_core` 3.7.1_0. Then it fails during the build of `py38-pgkutil_resolve_name` with the message `ERROR Missing dependencies: flit_core >=2,<3`. Suggestions to solve this are welcome!

Reply:
Changing the restriction for `flit_core>=2` in `pyproject.toml` seems to work – at least with Python 3.8.